app.controllers.js

```diff
@@ -43,14 +42,29 @@ const postCommentByArticleId = (req, res, next) => {
     const newComment = req.body;
     const {article_id} = req.params;
     
-    fetchArticleById(article_id).catch(next);
-        
-    addNewComment(newComment, article_id).then((comment) => {
+    fetchArticleById(article_id).then(() => {
+        return addNewComment(newComment, article_id)
+    })
+    .then((comment) => {
         res.status(201).send(comment);
     }).catch(err => {
         next(err);
     })
 }
 
+const patchVotesbyArticleId = (req, res, next) => {
+    const {article_id} = req.params;
+    const increaseVotes = req.body
+
+    fetchArticleById(article_id).then(() => {
```

Comment:
i would do this fetchArticleById in the model, keeping the model functionality and the controllers separate.